### ISSUE TITLE
error: The method 'startsWith' isn't defined for the type 'Object'. 

### ISSUE THREAD

**Issue by chenyongda2018** [pulyaevskiy/quill-delta-dart#16]
when I upgrade flutter sdk up to the newest 1.20,
```
Error: The method 'startsWith' isn't defined for the class 'Object'.
 - 'Object' is from 'dart:core'.
Try correcting the name to the name of an existing method, or defining a method named 'startsWith'.
      if (actualOp.data.startsWith('\n')) {
                        ^^^^^^^^^^


FAILURE: Build failed with an exception.
```

**Comment by pulyaevskiy** [comment]
Could you provide full stack trace? 

Also please see change log for 1.1.2. If you rely on operation value always being a string this is not going to work anymore, because operations can now store arbitrary objects. If you are not using embeds one your code then you can probably just add a type cast to String. Otherwise you'll need to check for the value to be a string.

**Comment by pulyaevskiy** [comment]
I just published 1.1.3 which reverts the recent change to value type and should make your errors go away.
I will be publishing it as a major version bump in 2.0.0.

For now just running flutter packages upgrade should resolve this issue for you.
